Source/ProgressionSystemRuntime/Public/Data/PSWorldSubsystem.h

```diff
@@ -110,6 +110,10 @@ class PROGRESSIONSYSTEMRUNTIME_API UPSWorldSubsystem : public UWorldSubsystem
 	UFUNCTION(BlueprintCallable, Category="C++")
 	float GetDifficultyMultiplier();
 
+	/** Returns spot component by the player tag, returns null if spot is not found */
```

Comment:
Please update comment as it does not use tags anymore

Reply:
done https://github.com/h4rdmol/ProgressionSystem/pull/40/commits/5e056a974c02e0b34b6224c53b909a4e6315daca